packages/fast-jss-manager-react/src/manage-jss.tsx

```diff
@@ -95,9 +67,9 @@ function manageJss<S, C>(styles?: ComponentStyles<S, C>): <T>(Component: React.C
             private static styleMap: WeakMap<ComponentStyles<S, C>, string> = new WeakMap();
 
             /**
-             * Store references to all separated stylesheets
+             * All static and dynamic styles associated with the component
```

Comment:
They are actually the styles of *all* component instances - I'm not sure you'll actually need this anymore though